frontend/src/pages/CalendarPage/components/Calendar/index.js

```diff
@@ -10,9 +10,29 @@ import {
   ViewSwitcher,
   TodayButton,
   DateNavigator,
+  Resources,
 } from '@devexpress/dx-react-scheduler-material-ui';
 import { ViewState } from '@devexpress/dx-react-scheduler';
 
+// color of events based on status
+const resources = [
+  {
+    fieldName: 'status',
+    instances: [
+      { id: 'complete', color: '#b3cce6' },
+      { id: 'pending', color: '#e6b3cc' },
+      { id: 'ready', color: '#f28c8c' },
+    ],
+  },
+];
+
+const mainResource = 'status';
+
+resources.propTypes = {
+  fieldName: PropTypes.string.isRequired,
+  instances: PropTypes.object.isRequired,
```

Comment:
Based on how you defined `instances` starting from line 21, the proptypes for it should be an array of objects, and not just an object

Reply:
Well actually, since resources is not even a prop of the Calendar component, you don't even need these propTypes definitions. Can probably just remove them. One day most of our files will be changed to TypeScript and we'll have a better time with types : )